pkg/constants/constants.go

```diff
@@ -379,6 +380,15 @@ const (
 	// is "Running", the cluster may not be in the same ready state as when it first finished
 	// installing. Use with caution.
 	ResumeSkipsClusterOperatorsLabel = "hive.openshift.io/resume-skips-cluster-operators"
+
+	// IBMCloudAPIKeySecretKey is a key used to store an api key token within a secret
+	IBMCloudAPIKeySecretKey = "ibmcloud_api_key"
+
+	// IBMCloudAPIKeyEnvVar is the name of the environment variable containing an IBM Cloud API key.
+	IBMCloudAPIKeyEnvVar = "IC_API_KEY"
+
+	// IBMCloudCredentialsSecretKey is a key used to store IBM environment variable credentials
+	IBMCloudCredentialsEnvSecretKey = "ibm-credentials.env"
```

Comment:
This can also wait for ManagedDNS for IBMCloud, or just left for future use.

Reply:
I left this one in.